downstream/modules/devtools/proc-devtools-extension-set-language.adoc

```diff
@@ -0,0 +1,43 @@
+[id="devtools-extension-set-language_{context}"]
+
+= Associating the Ansible language to YAML files
+
+[role="_abstract"]
+
+The Ansible {VSCode} extension works only when the language associated with a file is set to Ansible.
```

Comment:
Suggestion: I think you could wrap "Ansible" in bold to highlight it as a UI element.